notes-app/helper.js

```diff
@@ -1,56 +1,51 @@
 const fs = require('fs')
 const chalk = require('chalk');
+
 function getNotes () {
     try {
         const data = fs.readFileSync('notes.json')
-        const content = data.toString()
-        const jsonData = JSON.parse(content)
+        const jsonData = JSON.parse(data.toString())
         return jsonData
     }catch(err){
         console.log(chalk.red("err"))
        return []
     }
-    }
+}
 
 const saveNotes = (data)=>{
     const jsonData = JSON.stringify(data)
     fs.writeFileSync('notes.json',jsonData)
 }
+
 const addNotes=(title,body)=>{
-    const jsonData1 = getNotes()
-    const duplicate=jsonData1.find(x=>{if(x.title===title)return true})
-    console.log(duplicate)
-    if(!duplicate){
-    const jsonData =getNotes()
-    jsonData.push({"title": title, "body": body})
+    const jsonData = getNotes()
+    if(!jsonData.find(x=>{if(x.title===title)return true})){
+       jsonData.push({"title": title, "body": body})
         saveNotes(jsonData)
     }else{
         console.log("exist")
     }
 }
+
 const readNotes=(title)=>{
     const jsonData =getNotes()
     jsonData.forEach(x=>{if(x.title===title) console.log(chalk.red(x.title),chalk.green(x.body))})
```

Comment:
Your function should find note in `notes.json` and save this note in variable (e.g. const notes), then if your variable doesn't exist show message that note can't be found. To achieve this you should use another method neither `forEach` because `forEach` doesn't return anything, it only loops through array.